components/HTMLContent.js

```diff
@@ -26,6 +29,20 @@ export const isEmptyValue = value => {
   }
 };
 
+const getHTMLPreview = (html, characters = 50) => html.replace(/<\/?\w+>/g, '').slice(0, characters);
```

Comment:
This creates strange results:

![image](https://user-images.githubusercontent.com/1556356/93860741-cf97b300-fcbf-11ea-9217-75eacc4a6ae2.png)

![image](https://user-images.githubusercontent.com/1556356/93860776-dcb4a200-fcbf-11ea-88d1-6315071cc9be.png)

Some possible alternatives with the raw rich html content:
- Use CSS truncate (see https://github.com/opencollective/opencollective-frontend/blob/f80caaf53b977fcab1d500b4606895c43c0a1b09/components/Text.js#L20)
- Use a max height

Reply:
I updated that to return the first sentence from the HTML, otherwise, it would be impossible to reproduce what was requested in the design:
![image](https://user-images.githubusercontent.com/2119706/93919092-a2f59280-fce3-11ea-8453-4ba55be38c84.png)